galaxy.yml

```
@@ -2,13 +2,14 @@ namespace: ansible
 name: posix
 version: 0.1.0
 readme: README.md
-authors: null
+authors:
+  - Ansible (github.com/ansible)
 description: null
 license: GPL-3.0-or-later
 license_file: COPYING
 tags: null
 dependencies: {}
-repository: git@github.com:ansible-collection-migration/ansible.posix.git
-documentation: https://github.com/ansible-collection-migration/ansible.posix/tree/master/docs
-homepage: https://github.com/ansible-collection-migration/ansible.posix
-issues: https://github.com/ansible-collection-migration/ansible.posix/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc
+repository: https://github.com:ansible-collections/ansible.posix
+documentation: https://github.com/ansible-collections/ansible.posix/tree/master/docs
+homepage: https://github.com:ansible-collections/ansible.posix
+issues: https://github.com:ansible-collections/ansible.posix
```

For HTTP URLs you should replace the colon too. It's only needed for SSH
```suggestion
repository: https://github.com/ansible-collections/ansible.posix
documentation: https://github.com/ansible-collections/ansible.posix/tree/master/docs
homepage: https://github.com/ansible-collections/ansible.posix
issues: https://github.com/ansible-collections/ansible.posix
```